contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,162 @@
+<frontmatter>
+  title: Workflow Automation with Github Actions
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
```

Comment:
can add a table of contents here, refer to other PRs for examples

Reply:
Just curious, is there a real need for this aside from being consistent, since the right navbar doubles as a ToC?

Reply:
oh it will be responsive based on screen size, it will not show until ur screen is of a certain smaller size.